Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -408,7 +409,6 @@ using TransformOriginXY = Position;
 using TransformOriginY = PositionY;
 using TransformOriginZ = Length<>;
 using Transitions = CoordinatedValueList<Transition>;
-using WebkitBorderSpacing = Length<CSS::Nonnegative>;
```

Comment:
Let's keep this forward declared. If you need to change the range in two places, that's ok.